src/main/java/de/tum/in/www1/artemis/domain/assessment/dashboard/ExerciseRatingCount.java

```diff
@@ -0,0 +1,7 @@
+package de.tum.in.www1.artemis.domain.assessment.dashboard;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public record ExerciseRatingCount(Double averageRating, Long numberOfRatings) {
+}
```

Comment:
you could add a doc here as we do for new classes

Reply:
I can add this documentation in another PR 👍